crates/forge_domain/src/config.rs

```diff
@@ -0,0 +1,16 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Default, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ForgeConfig {
+    pub key_info: Option<ForgeKey>,
+}
+
+#[derive(Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct ForgeKey {
```

Comment:
Create a newtype called `ForgeKey(String)`. So that we don't work with strings instead have a type-safe version of it called `ForgeKey`.